cmd/submit.go

```diff
@@ -19,12 +19,14 @@ import (
 
 // submitCmd lets people upload a solution to the website.
 var submitCmd = &cobra.Command{
-	Use:     "submit FILE1 [FILE2 ...]",
+	Use: "submit [FILE1	 ...]",
```

Comment:
I wonder if we could somehow make this a bit more clear, where previously the text implied multiple files could be submitted. Nested square brackets is weird.

What about:

```suggestion
	Use: "submit [FILE1, ...]",
```
or 
```suggestion
	Use: "submit [FILE(S)]",
```

Reply:
```
ls [OPTION]... [FILE]...
```

From the man page for `ls` to indicate optional options and files, as an example of documentation...

Git uses:

```
command [<file> ...]
```

as another example for when a file or files are optionally given, and `less` uses

```
[filename] ...
```

to indicate 0 or more file names.

Reply:
Maybe we use the git version?

```suggestion
	Use: "submit [<FILE> ...]",
```

Reply:
@andrerfcsantos what do you think about that?